src/containerapp/azext_containerapp/_params.py

```diff
@@ -179,14 +187,20 @@ def load_arguments(self, _):
         c.argument('from_revision', help='Revision to copy from. Default: latest revision.')
         c.argument('image', options_list=['--image', '-i'], help="Container image, e.g. publisher/image-name:tag.")
 
+    with self.argument_context('containerapp revision label') as c:
+        c.argument('name', id_part=None)
+        c.argument('revision', help='Name of the revision.')
+        c.argument('label', help='Name of the label.')
+
     with self.argument_context('containerapp ingress') as c:
         c.argument('allow_insecure', help='Allow insecure connections for ingress traffic.')
         c.argument('type', validator=validate_ingress, arg_type=get_enum_type(['internal', 'external']), help="The ingress type.")
         c.argument('transport', arg_type=get_enum_type(['auto', 'http', 'http2']), help="The transport protocol used for ingress traffic.")
         c.argument('target_port', type=int, validator=validate_target_port, help="The application port used for ingress traffic.")
 
     with self.argument_context('containerapp ingress traffic') as c:
-        c.argument('traffic_weights', nargs='*', options_list=['--traffic-weight'], help="A list of revision weight(s) for the container app. Space-separated values in 'revision_name=weight' format. For latest revision, use 'latest=weight'")
+        c.argument('revision_weights', nargs='+', options_list=['--revision-weight'], help="A list of revision weight(s) for the container app. Space-separated values in 'revision_name=weight' format. For latest revision, use 'latest=weight'")
```

Comment:
Could we use the `deprecate_info` flag to the `traffic_weights` parameter to set deprecation information first to avoid directly causing breaking change to users?

Reply:
@zhoxing-ms essentially this renamed to `--revision-weight` to reduce confusion with the new parameter `--traffic-weight`. To get around the need for a deprecation, I can add `--traffic-weight` as an option for `--revision-weight`

Reply:
I suggest using the `deprecated_info` for `--traffic-weight` if the `--traffic-weight` parameter is no longer recommended in the future, what do you think of it?

Reply:
Sounds good I just added as a deprecated argument option